issue_release_status.go

```diff
@@ -99,13 +106,24 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 
 	logger.Info().Msgf("Handling deliveryID: '%s', eventType '%s'", deliveryID, eventType)
 
-	prBase := event.GetPullRequest().GetBase().GetRef() // The branch which the pull request is ending.
-	serviceName := event.GetRepo().GetName()
-	serviceName = strings.TrimSuffix(serviceName, "-service")
-	serviceName = strings.TrimPrefix(serviceName, "lunar-way-")
+	// Get info from event
+	prBase := event.GetPullRequest().GetBase().GetRef()
 	policyPath := handler.releaseManagerURL + "/policies?service="
 	describeArtifactPath := handler.releaseManagerURL + "/describe/artifact/"
 
+	// Get service name
+	var serviceName string
+	if handler.repoToServiceMap != nil {
+		_, ok := handler.repoToServiceMap[event.GetRepo().GetName()]
+		if ok == true {
+			serviceName = handler.repoToServiceMap[event.GetRepo().GetName()]
+		} else {
+			serviceName = trimServiceName(event.GetRepo().GetName())
+		}
+	} else {
+		serviceName = trimServiceName(event.GetRepo().GetName())
+	}
```

Comment:
The logical flow here is a bit hard to understand. If we extract it into a function it might become a bit more readable? What do you think?

The reasoning is: if we have a mapper and it has a hit, use that. Otherwise just trim the name.

```go
func serviceName(repoName string, mapping map[string]string) string {
	if handler.repoToServiceMap != nil {
		serviceName, ok := handler.repoToServiceMap[repoName]
		if ok {
			return serviceName
		}
	}
	return trimServiceName(repoName)
}
```

Reply:
Yeah that was a good idea 🚀 https://github.com/lunarway/release-manager-bot/pull/23/commits/2227e2294fc7579143f51510b37b559e5c352784